src/main/java/gui/TopPane.java

```diff
@@ -0,0 +1,43 @@
+package gui;
+
+import javafx.geometry.Insets;
+import javafx.scene.control.Button;
+import javafx.scene.control.Menu;
+import javafx.scene.control.MenuBar;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+
+/**
+ * Class that represents the whole of top-level elements in the gui.
+ * In other words, the file-edit-view-help menus and any buttons below that.
+ */
+class TopPane extends VBox {
```

Comment:
Shouldn't this be called the menu or the header? It is not a full size pane like the rest of the VBoxes will end up.